services/persistence/main.py

```diff
@@ -73,57 +142,48 @@ def create_manifesto_storage(texts, labels):
             )
         '''
     )
-
-    pks = []
-    for text in texts:
-        c.execute(
-            "INSERT INTO texts(statement) VALUES (?)",
-            (text, )
-        )
-        pks.append(c.lastrowid)
-
-    for text_id, label in zip(pks, labels):
-        c.execute(
-            """
-            INSERT INTO labels (texts_id, label, source) VALUES
-            (?, ?, ?)""",
-            (text_id, label, 'manifesto_project')
-        )
-
     conn.commit()
-
-    for statement, label, source, text_created, label_created in c.execute(
-            """
-            SELECT
-                t.statement,
-                l.label,
-                l.source,
-                t.created_at,
-                l.created_at
-            FROM texts t
-            INNER JOIN labels l ON l.texts_id = t.id
-            LIMIT 10"""
-    ):
-        print(statement, label, source)
-
-    print(
-        'inserted',
-        c.execute("SELECT count(1) FROM texts").fetchone(),
-        'texts with',
-        c.execute("SELECT count(1) FROM labels").fetchone(),
-        'labels'
-    )
-
     conn.close()
 
+    insert_into(DB_FILENAME, texts, labels, 'manifesto')
```

Comment:
das macht jetzt bulk inserts mit autoincrement keys? cool!

Reply:
Naja nicht so richtig Bulk, die Inserts sind aber nicht wirklich teuer und viele von denen machen wir nur einmal zu Beginn des Projekts.

Sauberer wäre es nochmal wenn wir die Inserts in einer Transaktion machen natürlich